3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,129 @@
+/** Smart-ease - Goes Global! **/
+
```

Comment:
this should be `gender` with small g. Also consider using default values, so you don't have to defined them in each call. For example the parameters could look like this:
````
function getFullname(firstName, surName, gender="male",  useFormalName=False) {
...
}

```
Example use: 
````
const fullname3 = getFullname("Banjamin", "Hughes", "female");
const fullname4 = getFullname("Banjamin", "Hughes", "female", true);
const fullname1 = getFullname("Banjamin", "Hughes");
const fullname2 = getFullname("Banjamin", "Hughes", "male", true);